ranking/opts.go

```diff
@@ -57,10 +57,14 @@ func boolFromQuery(query url.Values, name string) bool {
 }
 
 func addSuffixesForFiletype(suffixes *map[string]float32, filetype string) {
+	// what exactly happens here? what does the number assign do?
+	// i'm just trying to add support for Objective-C source files
 	switch filetype {
 	case "c":
 		(*suffixes)[".c"] = 0.75
 		(*suffixes)[".h"] = 0.75
+	case "m":
+		(*suffixes)[".m"] = 0.75
```

Comment:
In https://anonscm.debian.org/cgit/pkg-gnustep/volumecontrol.app.git/tree/ I indeed see a .h file. So, that should be included in your pull request I assume.